s3/replication/common/tests/system/config/config.yaml

```diff
@@ -9,9 +9,10 @@ object_tag_name: user-tag
 object_tag_value: user-tag-value
 range_read_offset: -1
 range_read_length: -1
-object_size: 1024
+object_size: 26214400
 total_objects: 100
 transfer_chunk_size: 4096
+total_parts: 5
```

Comment:
`# Make sure object_size / total_parts >= 5Mb for all parts.`

Reply:
done